### ISSUE TITLE
[BUG] \Phalcon\Session\Flash::getMessages();

### ISSUE THREAD

**Issue by tugrul** [phalcon/cphalcon#11941]
## Test
```php
$flash = new \Phalcon\Session\Flash();
$flash->error('Error Message');
var_dump($flash->getMessages('success', false));
```
## Expected Result
```
array (size=0)
  empty
```
## Actual Result
```
array (size=1)
  'error' => 
    array (size=1)
      0 => string 'Error Message' (length=13)
```

**Comment by stamster** [comment]
This issue is present only in 2.1.x branch?

**Comment by sergeyklay** [comment]
Fixed in the `2.1.x` branch.
